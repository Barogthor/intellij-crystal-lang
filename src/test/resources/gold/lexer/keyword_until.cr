until some_condition
  do_this
end