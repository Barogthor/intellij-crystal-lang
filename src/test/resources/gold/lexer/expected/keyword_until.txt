until ('until')
WHITE_SPACE (' ')
IDENT ('some_condition')
WHITE_SPACE ('\n  ')
IDENT ('do_this')
WHITE_SPACE ('\n')
end ('end')